test/e2e/test/elasticsearch/steps_mutation_test.go

```diff
@@ -22,13 +23,13 @@ func Test_clusterUnavailability(t *testing.T) {
 	require.False(t, cu.hasExceededThreshold())
 
 	// mark the cluster as available, we're still below the threshold
-	cu.markUnavailable()
+	cu.markUnavailable(errors.New("Connection refused"))
```

Comment:
It doesn't matter here but error strings should not be capitalized (unless they start with proper names or acronyms). To keep the habit?

```suggestion
	cu.markUnavailable(errors.New("connection refused"))
```